mergify_engine/queue/merge_train.py

```diff
@@ -638,6 +638,23 @@ async def update_summaries(
         queue_summary = "\n\nRequired conditions for merge:\n\n"
         queue_summary += evaluated_queue_rule.conditions.get_summary()
 
+        if self.failure_history:
+            batch_failure_summary = f"\n\nThe pull request {self._get_user_refs()} is part of previously a failed speculative checks batch:\n"
```

Comment:
```suggestion
            batch_failure_summary = f"\n\nThe pull request {self._get_user_refs()} is part of a previously failed speculative checks batch:\n"
```

Reply:
Actually, since you need to fix it again, make it easier to read:
```suggestion
            batch_failure_summary = f"\n\nThe pull request {self._get_user_refs()} is part of a speculative checks batch that previously failed:\n"
```